model/rulefmt/rulefmt.go

```diff
@@ -233,6 +233,14 @@ func (r *RuleNode) Validate() (nodes []WrappedError) {
 				node: &r.Record,
 			})
 		}
+		// While record is a valid UTF-8 it's common mistake to put PromQL expression in the record name.
+		// Disallow "{}" chars.
```

Comment:
Not sure if this is worth it. People will still make this mistake with expressions that do not contain `{` or `}`, e.g. `sum(rate(requests_total[5m]))`.

I was thinking we could ask for the same syntax as in PromQL, i.e. a name with non-standard characters has to be written like `{"my.non.standard.name"}`, but this might lead to Yaml quoting hell… WDYT?

Reply:
Yea, fair. I think it's easier to relax later than to restrict later 🤔  Not sure, probably cleanest is to not validate this and just allow for those mistakes.